tests/test_labgrid_resources.py

```diff
@@ -9,15 +10,15 @@
 # resources on the fly.
 
 
+@pytest.mark.slow
 def test_labgrid_resources_simple(strategy, shell, check):
     """Test non-managed resources."""
 
-    def retry_loop():
+    def retry_loop(logger):
         rpm = RemotePlaceManager.get()
         exporter = strategy.target_hostname
 
-        for _ in range(300):
-            time.sleep(1)
+        for _ in range(int(300 / 15)):
```

Comment:
Use integer division:

```suggestion
        for _ in range(300 // 15):
```

Reply:
Thx. (I am a little surprised that ruff did not suggest this change.)